tests/helper/helper_interactive.go

```diff
@@ -57,7 +57,8 @@ func RunInteractive(command []string, env []string, tester Tester) (string, erro
 
 	term := vt10x.New(vt10x.WithWriter(pts))
 
-	c, err := expect.NewConsole(expect.WithStdin(ptm), expect.WithStdout(term), expect.WithCloser(pts, ptm), expect.WithDefaultTimeout(3*time.Minute))
+	// Set to 1 MINUTE timeout, since the command may take a while to start
+	c, err := expect.NewConsole(expect.WithStdin(ptm), expect.WithStdout(term), expect.WithCloser(pts, ptm), expect.WithDefaultTimeout(1*time.Minute))
```

Comment:
Wouldn't this be too short as a timeout, because `helper.RunInteractive` is called in few other tests that run `dev`, `deploy`. Those commands might take longer, no?
#5518 previously set this to 3 minutes, as a default timeout. But I am not sure how reducing this to 1 minute would impact the tests in CI.

Reply:
I've changed it back to 3 minutes now